pandas/tests/io/parser/test_encoding.py

```diff
@@ -5,6 +5,7 @@
 
 from io import BytesIO
 import os
+import tempfile
```

Comment:
can pytest builtin fixture or ensure_clean be used instead?

Reply:
I need the `encoding` to be parameterized in this test, which can't be done with `ensure_clean` at the moment (though could be a useful enhancement as a follow-up).

The `pytest` builtin fixture has a similar issue (as an aside, if I had to choose between `ensure_clean` or `pytest` fixture, I would generally go with our in-house one since it's a little more flexible to use).